pkg/frameworkPackage.targets

```diff
@@ -30,55 +41,86 @@
 
     <Dependency Include="Microsoft.NETCore.Platforms">
       <Version>$(PlatformPackageVersion)</Version>
+      <TargetFramework>$(TargetFramework)</TargetFramework>
     </Dependency>
   </ItemGroup>
 
   <!-- Bring in lib content from binplaced lib props -->
-  <Import Condition="'$(PackageTargetRuntime)' != ''" Project="$(LibBinDir)\*.props" />
-  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
+  <Import Condition="'$(IncludeLibFiles)' == 'true'" Project="$(LibBinDir)\*.props" />
+  <ItemGroup Condition="'$(IncludeLibFiles)' == 'true'">
     <!-- Include lib -->
-    <File Include="@(FileToPackage)">
-      <TargetPath Condition="'%(File.TargetPath)' == ''">runtimes/$(PackageTargetRuntime)/lib/$(TargetFramework)</TargetPath>
+    <File Include="@(LibFile)">
+      <TargetPath Condition="'%(File.TargetPath)' == ''">$(LibFileTargetPath)</TargetPath>
     </File>
     <!-- force a missing file if lib build is absent -->
-    <File Include="$(LibBinDir)/MISSING_LIB_BUILD" Condition="'@(FileToPackage)' == ''" />
+    <File Include="$(LibBinDir)/MISSING_LIB_BUILD" Condition="'@(LibFile)' == ''" />
+  </ItemGroup>
 
+  <ItemGroup Condition="'$(IncludeNativeFiles)' == 'true'">
     <!-- Include native -->
     <ExcludeNative Include="$(NativeBinDir)/*.lib" />
     <NativeFile Include="$(NativeBinDir)/*.*" Exclude="@(ExcludeNative)">
-      <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
+      <TargetPath>$(NativeFileTargetPath)</TargetPath>
     </NativeFile>
     <File Include="@(NativeFile)" />
     <!-- force a missing file if native build is absent -->
     <File Include="$(NativeBinDir)/MISSING_NATIVE_BUILD" Condition="'@(NativeFile)' == ''" />
   </ItemGroup>
 
-
-  <Target Name="VerifyClosure" AfterTargets="Build">
+  <Target Name="_getFinalFilesExisting">
     <ItemGroup>
-      <_fileExisting Include="@(File)" Condition="Exists(%(FullPath))"/>
+      <_refFileExisting Include="@(File)" Condition="Exists('%(FullPath)') AND ($([System.String]::new('%(File.TargetPath)').StartsWith('ref')) OR '%(File.TargetPath)' == '$(RefFileTargetPath)')" />
+      <_refFileNames Include="@(_refFileExisting -> '%(FileName)')">
+        <Original>%(_refFileExisting.Identity)</Original>
+      </_refFileNames>
+      <_refFileNamesFiltered Include="@(_refFileNames)" Exclude="@(ExcludeFromClosure)" />
```

Comment:
>ExcludeFromClosure [](start = 67, length = 18)

Just wondering if we should have an Exclude Item for refs and exclude item from runtime.